### PR TITLE
CORTX-32897: Support default parameters for pre-merge job

### PR DESCRIPTION
**Pull request by rkothiya**
Since Motr-Hare-Single-Node-Cluster-Deployment-On-Rocky-Linux
will be used as a pre merge job we need to remove the parameters
that it accepts and run the job on dedicated machines.

Signed-off-by: Rinku Kothiya <rinku.kothiya@seagate.com>

# Problem Statement
Cannot give parameters to pre merge job Motr-Hare-Single-Node-Cluster-Deployment-On-Rocky-Linux.

# Design
Modified the job to run on dedicated machines.

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by rkothiya** [comment]
retest this please

File: scripts/jenkins/motr-hare-1node-cluster.groovy [hunk 1]
```diff
@@ -1,56 +1,55 @@
...
-                    sshpass -p ${NODE_PASS} ssh -o StrictHostKeyChecking=no ${NODE_USER}@${NODE_HOST} "/opt/seagate/cortx/hare/libexec/m0crate-io-conf > /root/crate.yaml"
-                    sshpass -p ${NODE_PASS} ssh -o StrictHostKeyChecking=no ${NODE_USER}@${NODE_HOST} "/root/cortx-motr/utils/m0crate -S /root/crate.yaml"
+                sh label: 'Download code and install single node cluster', script: '''
+                    hostname
+                    pwd
+                    ln -s "$WORKSPACE" ../cortx-motr
+                    ./scripts/install/usr/libexec/cortx-motr/motr-cleanup || true
+                    losetup -D
+                    ./scripts/build-prep-1node.sh -dev
+                    hctl bootstrap --mkfs ../singlenode.yaml
```
Comment (madhavemuri) [inline scripts/jenkins/motr-hare-1node-cluster.groovy:28]: Add `hctl status -d` and make sure all ioservice are online before doing the IO.

Comment (rkothiya) [reply scripts/jenkins/motr-hare-1node-cluster.groovy:28]: I have added the "hctl status -d". 
If the ioservice are not online then the job will fail and we should be able to see that in the status.

Comment (madhavemuri) [reply scripts/jenkins/motr-hare-1node-cluster.groovy:28]: Add a sleep of few seconds and try it one more time, then check status again and then only fail it.

Comment (rkothiya) [reply scripts/jenkins/motr-hare-1node-cluster.groovy:28]: I have put a sleep of 60 seconds.

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: scripts/jenkins/motr-hare-1node-cluster.groovy (hunk 1 above)
Comment (rkothiya) [reply scripts/jenkins/motr-hare-1node-cluster.groovy]: done

**Review by madhavemuri — APPROVED** [review]
Can you remove the reference of Rock Linux from the commit message, in future we may use any OS with the same job.

**Comment by rkothiya** [comment]
As this is a stand alone script and ci/cd job do not test this and also it does not have anything to do with deployment hence giving an exception for the pre-merge jobs.

**Pull request merged**